app/templates/content.hbs

```diff
@@ -88,6 +88,23 @@
                     {{/if}}
 
                     <div class="p-t-xs">
+                        <h4 class="p-v-md f-w-md">{{t "global.license"}}</h4>
+                        <p class="p-v-md p-b-sm">
+                            <span style='cursor: pointer'>
+                                {{#if showLicenseText }}
+                                    {{fa-icon 'caret-down' click=(action 'toggleLicenseText')}}
+                                {{else}}
+                                    {{fa-icon 'caret-right' click=(action 'toggleLicenseText')}}
+                                {{/if}}
+                            </span>
+                            {{model.license.name}}
```

Comment:
Can you move the license name back to before the caret? Sorry I wasn't clear, i just meant change the caret directions, but I think the caret still looks better after the license name.